src/scripts/main.js

```diff
@@ -1 +1,25 @@
 'use strict';
+
+document.querySelector('DOMContentLoaded', () => {
```

Comment:
The event 'DOMContentLoaded' should be passed to 'addEventListener', not 'querySelector'. Use 'document.addEventListener('DOMContentLoaded', () => {...})' instead.